src/server.imba

```diff
@@ -38,7 +38,15 @@ srv.get(/^([^\.]*)$/) do |req,res|
 			<meta name="keywords" content="imba">
 			<link href='http://fonts.googleapis.com/css?family=Source+Code+Pro:400,500,600' rel='stylesheet' type='text/css'>
 			<link rel="stylesheet" href="/dist/main.css" media="screen">
-			<style html=highlighter:theme.CSS>
+			<style html=highlighter:theme.CSS>		
+			# <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png?v=2">
+			# <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png?v=2">
+			# <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png?v=2">
+			# <link rel="manifest" href="/favicon/site.webmanifest?v=2">
+			# <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg?v=2" color="#191a25">
+			# <link rel="shortcut icon" href="/favicon/favicon.ico?v=2">
+			# <meta name="msapplication-TileColor" content="#191a25">
```

Comment:
I don't see any favicon added? So it won't load. You could create a directory favicon in `www/favicon` and put the files there.